src/refactorings/change-signature/change-signature.test.ts

```diff
@@ -0,0 +1,404 @@
+import { Code, ErrorReason, RelativePath } from "../../editor/editor";
+import { InMemoryEditor } from "../../editor/adapters/in-memory-editor";
+import { testEach } from "../../tests-helpers";
+
+import { changeSignature } from "./change-signature";
+
+type TestSample = {
+  code: Code;
+  path: RelativePath;
+};
+
+describe("Change Signature", () => {
+  testEach<{
+    setup: { currentFile: Code; path: RelativePath };
+    expected: { currentFile: Code };
+  }>(
+    "In same file",
+    [
+      {
+        description: "when there are a function without references",
+        setup: {
+          currentFile: `function [cursor]add(a, b) {
+            return a + b;
+          }`,
+          path: new RelativePath("./aFileWitoutReferences.ts")
```

Comment:
🍰 For these tests, since everything is supposed to be in the same file, I think we can omit the `path` attribute altogether and just make the test body use the same path to write the code and read it for asserts. What do you think?

Reply:
So it would be a regular `code: string; expected: string` instead of a `setup/expected`

Reply:
Yeah!!
With this changes test cases are much simpler 

https://github.com/nicoespeon/abracadabra/pull/725/commits/05cf6cdf42fbd4ff3cd4c2ed92edb8968aee0961